Description: relax dependency on log
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-02-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,7 +16,7 @@
 ignore = "0.4.18"
 indexmap = { version = "1.7", features = ["serde"] }
 itertools = ">= 0.9.0, < 0.11.0"
-log = "0.4.14"
+log = "0.4.11"
 md5 = "0.7.0"
 path-clean = "0.1.0"
 pretty_assertions = "1.1.0"
